<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableComparisonForm</class>
 <widget class="QWidget" name="TableComparisonForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QToolButton" name="btnSave">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Save SQL to a file</string>
       </property>
       <property name="statusTip">
        <string>Save SQL to a file</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="dbm.qrc">
         <normaloff>:/images/actions/images/actions/filesave.png</normaloff>:/images/actions/images/actions/filesave.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnInject">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Inject this SQL to a database and create the structures defined</string>
       </property>
       <property name="statusTip">
        <string>Inject this SQL to a database and create the structures defined</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="dbm.qrc">
         <normaloff>:/images/actions/images/small/database.png</normaloff>:/images/actions/images/small/database.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="grpLeft">
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="lblFrom">
             <property name="text">
              <string>From</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QComboBox" name="cmbVersionLeft"/>
             </item>
             <item>
              <widget class="QLineEdit" name="txtTab1"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTreeWidget" name="treeWidgetLeft">
           <property name="rootIsDecorated">
            <bool>true</bool>
           </property>
           <property name="allColumnsShowFocus">
            <bool>true</bool>
           </property>
           <property name="headerHidden">
            <bool>false</bool>
           </property>
           <attribute name="headerDefaultSectionSize">
            <number>50</number>
           </attribute>
           <column>
            <property name="text">
             <string>Section</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>PK</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Type</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grpRight">
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="lblTo">
             <property name="text">
              <string>To</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QComboBox" name="cmbVersionRight"/>
             </item>
             <item>
              <widget class="QLineEdit" name="txtTab2"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTreeWidget" name="treeWidgetRight">
           <property name="rootIsDecorated">
            <bool>true</bool>
           </property>
           <property name="allColumnsShowFocus">
            <bool>true</bool>
           </property>
           <property name="headerHidden">
            <bool>false</bool>
           </property>
           <attribute name="headerDefaultSectionSize">
            <number>50</number>
           </attribute>
           <column>
            <property name="text">
             <string>Section</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>PK</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Type</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="font">
      <font>
       <family>Courier 10 Pitch</family>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="dbm.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnInject</sender>
   <signal>clicked()</signal>
   <receiver>TableComparisonForm</receiver>
   <slot>onInject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>TableComparisonForm</receiver>
   <slot>onSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onInject()</slot>
  <slot>onSave()</slot>
 </slots>
</ui>
